Mathlib/CategoryTheory/Limits/Final.lean

```diff
@@ -306,9 +306,21 @@ variable (G)
 
 https://stacks.math.columbia.edu/tag/04E7
 -/
+@[simps!]
```

Comment:
I am not sure we want this definition to unfold automatically. For now, I would suggest not declaring `_hom` and `_inv` as simp lemmas. In the future, we may add a few equational lemmas:
```suggestion
@[simps! (config := .lemmasOnly)]
```
(Same for `limitIso`.)
Note: could you remove the definitions `colimitIso'` and `limitIso'` which seem unused.